src/components/styles/__snapshots__/Styles.test.tsx.snap

```diff
@@ -1,3 +1,3 @@
+exports[`allows to set global generic styles  1`] = `undefined`;
```

Comment:
should this be undefined?

Reply:
seems like this could cause a few breaking changes

Reply:
the problem is in our custom `getGlobalStyleTags` function. Apparently it doesn't get the styles anymore. It seems like the `document` is empty for some reason. I test it manually and the global styles work as expected